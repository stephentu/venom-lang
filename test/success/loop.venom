i = 0;
while i < 10 do
  x = i + 5;
  if x > 10 then
    print (x);
    print ("x > 10");
  else
    print (x);
    print ("x <= 10");
  end
  i = i + 1;
end
