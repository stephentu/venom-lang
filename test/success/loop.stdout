5
x <= 10
6
x <= 10
7
x <= 10
8
x <= 10
9
x <= 10
10
x <= 10
11
x > 10
12
x > 10
13
x > 10
14
x > 10
